### ISSUE TITLE
Direct Video VGA converter forced_scandoubler not working and stuck at 15khz

### ISSUE THREAD

**Issue by SUPER-DAN** [MiSTer-devel/PSX_MiSTer#213]
Hi, 

When using a VGA converter (using direct_video=1) with Scan Doubler enabled (forced_scandoubler=1) the output is not 31khz, but stuck at 15khz. No output, as soon as we start the core, VGA CRT monitors says : out of range 15.7Khz not supported.

The core seems to bypass what is set in the Mister ini file.  I also forced the resolution (always 60hz and supported resolution by the monitors). Tried multiple VGA CRT monitors, same thing.  Also tried another Mister same issue.

Also the mention; all other cores works fine and double the frequency (15.7Khz to 31Khz). 

Thanks in advance!

**Comment by RobertPeip** [comment]
The core has no scandoubler, as it's outputting all kind of resolutions (and interlaced) with different pixel clock that won't work on the frameworks scandoubler.

The only thing you can do is use the VGA scaler feature.

**Comment by SUPER-DAN** [comment]
Thanks Robert, I am not too sure how to setup VGA scaler.  I tried and it always go back to 15.7 Khz.  Would be great to have a tutorial for this if one day you have the time.  Thanks

**Comment by birdybro** [comment]
Here are all the relevant docs that might help.

https://mister-devel.github.io/MkDocs_MiSTer/advanced/ini/#analog-video-settings

https://mister-devel.github.io/MkDocs_MiSTer/advanced/crttable/

https://mister-devel.github.io/MkDocs_MiSTer/advanced/crt/

https://mister-devel.github.io/MkDocs_MiSTer/advanced/directvideo/

vga_scaler makes the vga output of the analog IO board output the same signal as HDMI does. I'm not sure if the direct video adapter supports it, I think only forced_scandoubler is supported as an option for direct video.

**Comment by SUPER-DAN** [comment]
I hate to give up, but I read every single articles, seek for help on forums, and I cannot output a 31khz frequency signal over the HDMI to VGA adapter using the scaling method.  My VGA CRT monitor always indicate 15.7KHZ for the PSX core, no matter what I tried.  All other cores are fine.  So the VGA scaling seems to be ignored no matter what I put. 

I also try 120Hz scaling method (to double 240p / 480i) with the setting below, no go.  Not sure what to do...

Video Mode= 2048,40,442,200,240,3,10,8,85909
VGA Scaler= 1
Vscale= 1
Vsync= 0

**Comment by SUPER-DAN** [comment]
Alright, I got it to work and I think this is an issue actually in either Mister or PSX core.  Let me explain and it's two things.

To get it to work(to output at 31KHZ, I had to disabled direct_video (=0) with a resolution higher than twice the 478i (the one that worked 1280x1024@60, no other resolution worked).  This was the secret sauce right down below :

[PSX]
forced_scandoubler=0
vga_scaler=1
video_mode=4
vscale_mode=1
direct_video=0
